s3tests/functional/test_s3.py

```diff
@@ -1489,10 +1496,13 @@ def test_object_set_get_metadata_none_to_good():
     eq(got, 'mymeta')
 
 
+# NOTE: skipping metadata related tests below b/c s2 only uses metadata
```

Comment:
Please change `s2` to SmartStore or Splunk

Reply:
will do